src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/BurdenEstimates/BurdenEstimateTests.kt

```diff
@@ -41,6 +41,8 @@ abstract class BurdenEstimateTests : DatabaseTest()
     {
         val returnedIds = setUp(db)
         TestUserHelper.setupTestUser()
+
+        db.addExpectations(returnedIds.responsibilityId)
```

Comment:
Now that we're retrieving expectations when populating the estimates, we need to add this to the test setup